guide/PrinciplesGuidelines.md

```diff
@@ -0,0 +1,39 @@
+## Community Principles
+
+These 5 principles can be used to evaluate a component’s fit for TELUS Design System community components.
+
+1. Must follow [digital design principles](https://docs.google.com/presentation/d/1C1pyN2dfaIYOz0_3SscfoDcbkbIIGLecWZorBB-SfU8/edit#slide=id.g335f368764_1_799)
```

Comment:
The `tds-community` repo is public facing. Should we point to a resource on [](https://tds.telus.com/) instead of Google Docs?